### ISSUE TITLE
NVDA is not reading label in Chrome when HTML has certain layout with custom elements

### ISSUE THREAD

**Issue by stormoz** [nvaccess/nvda#8676]
We have hit a weird issue with NVDA failing to read labels or button text of HTML pages that have certain layout in Chrome. We are building a web application with Angular that renders components as custom HTML elements by default. Somehow certain order/layout of custom elements and input elements/buttons causes NVDA to stop working properly in Chrome. Edge and Firefox are working as expected.

### Steps to reproduce:
- Open an html file with the following HTML in Chrome:
```
<!DOCTYPE html>
<html lang="en">
<head><title>NVDA is not reading label in Chrome</title></head>
<body>
	<custom-one><!-- replacing custom elements with DIVs fixes it -->
		<custom-two>
			<div></div>
		</custom-two><!-- or just adding a line break here fixes it --><div></div>
		<label for="test">Hello</label>
		<input id="test" type="text" />
	</custom-one>
</body>
</html>
```
- Focus on the input element

### Actual behavior:
- NVDA is not reading the label when focus is set on the text input element

### Expected behavior:
- NVDA is expected to read the label and say "Hello edit..." every time a focus is set on the input element

#### NVDA Installed/portable/running from source:
- Installed

#### NVDA version:
- 2018.2.1

#### Windows version:
- Windows 8.1 Enterprise, x64 (6.3.9600 Build 9600)

#### Name and version of other software in use when reproducing the issue:
- Chrome 67.0.3396.62 (Official Build) (32-bit)

#### Other information about your system:
- N/A

#### Does the issue still occur after restarting your PC?
- Yes

#### Have you tried any other versions of NVDA?
- Yes, the same behaviour with a version of NVDA from mid 2016

Log:
[log.txt](https://github.com/nvaccess/nvda/files/2322035/log.txt)

**Comment by scottaohara** [comment]
Could this be a Chrome bug rather than NVDA specific, since it works as expected in Firefox and Edge?

I did a quick test with Safari & Chrome with VoiceOver on macOS.  The label was announced with Safari + VO, but not when using Chrome.

**Comment by brayanGuarnizo** [comment]
I encountered the same problem with Google Chrome Version 75.0.3770.142 (Offizieller Build) (64-Bit) is there any solution for this?  
For any help. I will be grateful

**Comment by Adriani90** [comment]
It seems this issue occurs only when routing the mouse to the edit field or when pressing tab while in browse mode. When I enable focus mode with nvda key + space and navigate with tab, moving the system focus, the label is read properly.
Testing with NVDA 2019.3 Beta 1 on Chrome 79.
cc: @derekriemer do you have any hints for this?

**Comment by Adriani90** [comment]
This is actually working as expected now in all browsers. The only exception is when putting the focus on the edit field with the mouse, the label is not reported. This is because the label itself is on the left side of the edit field and probably not programatically connected to each other.

I am closing this as works for me.
A user using the mouse would notice there is a label on the edit field and this will be reported as soon as the mouse is moved to the left of the edit field.
